server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -386,7 +386,7 @@ private void registerAsyncMetrics(MeterRegistry registry) {
 
         metrics.add(
             registry.registerLongGauge(
-                "es.node.stats.indices.indexing.memory.limit.current",
+                "es.node.stats.indices.indexing.memory.limit.total",
```

Comment:
Same: just `limit` (and it needs to be added to the suffixes too probably)

Reply:
how about memory_limit.total ?